lexicon/conversion.py

```diff
@@ -140,6 +140,69 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from a regular text file, where each line contains:
+    < WORD > < PHONEME1 > < PHONEME2 > ...
+    separated by tabs or spaces.
+    The lemmata will be added in the order they appear in the text file,
```

Comment:
Again: The code below assumes that the lines of the file are sorted alphabetically, right? (Because of the comparison with `last_lemma`.) If the file is not sorted alphabetically, there might be separate entries for the same lemma with different pronunciations which might or might not cause problems.